src/pageObjects/GaragePage/GaragePage.js

```diff
@@ -0,0 +1,31 @@
+import BaseComponent from '../../components/BaseCoomponent';
+
+export default class GaragePage extends BaseComponent {
+  _garagePageContentSelector = 'div.app-content';
+
+  _settingsBtnSelector = 'a[routerLink=settings]';
```

Comment:
I would create dedicated component NavBar fro navigation items

Reply:
Created NavBar pom and updated GaragePage pom